integration_tests/features/Reorgs.feature

```diff
@@ -91,3 +91,147 @@ Feature: Reorgs
     When I create a transaction TX2 spending CB1 to UTX2
     When I submit transaction TX2 to PNODE1
     Then PNODE1 has TX2 in MEMPOOL state
+
+Scenario Outline: Massive multiple reorg
+        #
+        # Chain 1a:
+        #   Mine X1 blocks (orphan_storage_capacity default set to 10)
+        #
+    Given I have a seed node SEED_A1
+        # Add multiple base nodes to ensure more robust comms
+    And I have a base node NODE_A1 connected to seed SEED_A1
+    And I have a base node NODE_A2 connected to seed SEED_A1
+    When I mine <X1> blocks on SEED_A1
+    Then node NODE_A1 is at height <X1>
+    Then node NODE_A2 is at height <X1>
+        #
+        # Chain 1b:
+        #   Mine Y1 blocks (orphan_storage_capacity default set to 10)
+        #
+    And I have a seed node SEED_A2
+        # Add multiple base nodes to ensure more robust comms
+    And I have a base node NODE_A3 connected to seed SEED_A2
+    And I have a base node NODE_A4 connected to seed SEED_A2
+    When I mine <Y1> blocks on SEED_A2
+    Then node NODE_A3 is at height <Y1>
+    Then node NODE_A4 is at height <Y1>
+        #
+        # Connect Chain 1a and 1b
+        #
+    And I connect node NODE_A1 to node NODE_A3 and wait 1 seconds
+    And I connect node NODE_A2 to node NODE_A4 and wait 1 seconds
+    And I connect node SEED_A1 to node SEED_A2 and wait <SYNC_TIME> seconds
+    Then node SEED_A1 is at the same height as node SEED_A2
+    When I mine 10 blocks on SEED_A1
+    Then node SEED_A2 is at the same height as node SEED_A1
+    Then node NODE_A1 is at the same height as node SEED_A1
+    Then node NODE_A2 is at the same height as node SEED_A1
+    Then node NODE_A3 is at the same height as node SEED_A1
+    Then node NODE_A4 is at the same height as node SEED_A1
```

Comment:
Would it not be better to replace this with the function: then all nodes are at the same height as X

Reply:
That is an approach I considered, but it (the _all nodes_ part) would not work for chain 2 though if someone changes the number of blocks that are mined, so to keep with the pattern, I added the explicit steps.